### ISSUE TITLE
Segmentation fault while exporting .inp mesh file

### ISSUE THREAD

**Issue by Hx83866** [rquey/neper#159]
Hello Everyone,

I got some problems with mesh module when I was trying to export the meshed RVE to .INP file. 
Here are my code to generate and mesh  the RVE: 

**tessellation**
```
neper -T -n 4::9::from_morpho\
-morpho "voronoi::diameq:lognormal(2.527,0.602),aspratio(1.9289,1)::lamellar(w=2.0)"\
-o sheet_05\
-dim 2\
-reg 1\
-rsel 3.0\
-domain "square(80,80)"
```
**mesh**
`neper -M sheet_05.tess -statelset diameq -cl 3.0 -dim 3 -format inp -o sheet_05`

and I got this when the software terminated itself:

```
========================    N   e   p   e   r    =======================
Info   : A software package for polycrystal generation and meshing.
Info   : Version 4.1.2-2
Info   : Built with: gsl|muparser|opengjk|openmp|nlopt
Info   : Running on 8 threads.
Info   : <https://neper.info>
Info   : Copyright (C) 2003-2020, and GNU GPL'd, by Romain Quey.
Info   : Ignoring initialization file.
Info   : ---------------------------------------------------------------
Info   : MODULE  -M loaded with arguments:
Info   : [ini file] (none)

Info   : [com line] sheet_05.tess -statelset diameq -cl 3.0 -dim 3
         -format inp -o sheet_05
Info   : ---------------------------------------------------------------
Info   : Reading input data...
Info   :   - Reading arguments...
Info   : Loading input data...
Info   :   - Loading tessellation...
Info   :     [i] Parsing file `sheet_05.tess'...
Info   :     [i] Parsed file `sheet_05.tess'.
Info   : Meshing...
Info   :   - Preparing... (cl = 3)
Info   :   - 0D meshing... 100%
Info   :   - 1D meshing... 100%
Info   :   - 2D meshing... 100% (0.002|0.82/97%| 3%| 0%)
Info   :   - 3D meshing... 100%
Info   : Searching nsets and fasets...
Info   : Writing mesh results...
Info   :   - Preparing mesh...
Info   :   - Mesh properties:
Info   :     > Node number:     2970
Info   :     > Elt  number:     8475
Info   :     > Mesh volume: 19202.789
Info   :   - Writing mesh...
Info   :     [o] Writing file `sheet_05.inp'...
Segmentation fault
```

However, if the `-format inp` argument was not given, I could still get a .msh file. (see below)
![sheet_05_mesh](https://user-images.githubusercontent.com/35709665/112371395-d8a12580-8cde-11eb-9897-f4180764f342.png)

Does anyone have some ideas why this problem came up?

Thx! 

Best Regards,
Xiang Hu

**Comment by fzw9381** [comment]
I encountered the same problem while using neper.

**Comment by ztzt19** [comment]
me too. How can I solve it?

**Comment by rquey** [comment]
This will be fixed in version 4.4.1, to appear shortly.

To future readers: please provide a minimal working example when reporting a bug.  This helps us, and the bug will be fixed faster. This bug can be reproduced using:
```
neper -T -n 1 -dim 2
neper -M n1-id1.tess -dim 3 -format inp
```
